src/redux/actions/samples/loadSamples.js

```diff
@@ -1,22 +1,79 @@
 import fetchAPI from 'utils/http/fetchAPI';
 import handleError from 'utils/http/handleError';
+import config from 'config';
+import { api } from 'utils/constants';
 
 import {
   SAMPLES_LOADED,
   SAMPLES_ERROR,
   SAMPLES_LOADING,
 } from 'redux/actionTypes/samples';
 
+const toApiV2 = (samples) => {
+  const apiV2Samples = [{ samples: {} }];
+
+  const buildApiv1FilesObject = (fileObject) => {
+    const fileNames = [];
+    const apiV1FileObject = {};
+
+    Object.keys(fileObject).forEach((key) => {
+      const fileName = fileObject[key].s3Path.split('/').pop();
+      fileNames.push(fileName);
+
+      apiV1FileObject[fileName] = {
+        path: fileObject[key].s3Path,
+        size: fileObject[key].size,
+        valid: fileObject[key].valid,
```

Comment:
We shouldn't use `valid`. 
Here you can just always write "valid" because if a file was created it means it already passed the validity checks. 
`valid` is a property that wasn't cleaned up when it became redundant.